### ISSUE TITLE
NTR: X-ray: periosteal elevation (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8951]
New term request X-ray: periosteal elevation (mayman_090320174637): Osteomyelitis; periostitis; osteitis; pseudo-osteomyelitis; periosteal cloaking; periosteal reaction, bone; periosteal new bone formation; periostosis  UMLS: 2203599

**Comment by pnrobinson** [comment]
@MickeySegal
added as
Periosteal elevation
HP:0034970

**Comment by pnrobinson** [comment]
@MickeySegal - thoughts on the above?

**Comment by MickeySegal** [comment]
[Subperiosteal bone formation HP:0031485](https://hpo.jax.org/app/browse/term/HP:0031485) seems like one of the possible subsets, but the more general imaging finding can be caused by a variety of processes, including infection and hematomas.  As an example, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7475123/ "area of periosteal elevation is seen, probably caused by blood".

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000588
- label: Periosteal elevation
- parents: [Abnormal periosteum morphology (HP:0030313)](https://hpo.jax.org/app/browse/term/HP:0030313)
- pmid: PMID:32923528
- def: Lifting or separation of the periosteum from the outer surface of a bone. Periosteal elevation is usually seen in benign process, frequently in cranioencephalic traumas. There is an elevation of the periosteum after blood, like in cephalohematomas.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8951
- synonyms:
